package.json

```diff
@@ -17,23 +17,25 @@
     "@babel/preset-env": "^7.22.20",
     "@babel/preset-react": "^7.22.15",
     "@babel/preset-typescript": "^7.23.0",
+    "@chromatic-com/storybook": "^1.9.0",
```

Comment:
Can this chromatic visual testing add-on `@chromatic-com/storybook` be removed, as we don't tend to bundle this as it has associated costs

Reply:
Done!